price/src/error.rs

```diff
@@ -0,0 +1,9 @@
+#[derive(thiserror::Error, Debug)]
+pub enum PriceError {
+    #[error("unsupported token type: {0}")]
+    UnsupportedTokenType(i32),
+    #[error("unable to fetch price")]
+    UnableToFetch,
```

Comment:
Is there an underlying error when UnableToFetch hits?

Reply:
Doesn't look like it, `get_price_no_older_than` just returns None afaict.